src/main/resources/api/speakers.json

```diff
@@ -61,6 +65,10 @@
           "links": {
             "self": "/v1/speakers/1/relationships/session",
             "related": "/v1/speakers/1/session"
```

Comment:
Use `sessions` instead of `session` because one speaker can have multiple sessions
Do it at other places also.